WalletWasabi.Tests/RegressionTests/BackendTests.cs

```diff
@@ -188,38 +191,50 @@ public async Task StatusRequestTestAsync()
 					times++;
 				}
 
-				using var client = new WasabiClient(new Uri(RegTestFixture.BackendEndPoint), null);
-				var response = await client.TorClient.SendAsync(HttpMethod.Get, request);
-				using (HttpContent content = response.Content)
+				// First request.
 				{
-					var resp = await content.ReadAsJsonAsync<StatusResponse>();
-					Assert.True(resp.FilterCreationActive);
-				}
+					using HttpResponseMessage? response = await BackendHttpClient.SendAsync(HttpMethod.Get, requestUri);
```

Comment:
I was trying to use this block pattern before in the code. For the first time, it seems reasonable, but later on, I noticed that it is very hard to read and maintain. I am against this pattern. You can add a number to the response or create a helper method, or separate the tests if possible.

Reply:
Updated.

Reply:
Elegant solution!